PP-4617 - Disable auto deploy

kbottla opened the pull request:
## WHAT
- Added a flag to deploy docs manually as we are not aiming to automate the process now

kbottla committed:
PP-4617 - Disable auto deploy

- Added a flag to deploy docs manually as we are not aiming to automate the process now

rhowe-gds commented on deploy.sh:
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-set -euo pipefail
+set -eo pipefail
```
Why remove `u`? looks like we don't expect any unbound variables in this script

kbottla replied:
It is an issue with virtualenv which is failing this script on `./venv/bin/activate: line 57: PS1: unbound variable`

rhowe-gds replied:
ugh, that's nasty.
in the case, I'd surround that line with
`set +u`
and
`set -u`

kbottla replied:
👍 Updated to have it only around `./venv/bin/activate`

kbottla committed:
PP-4617 Allow unbound variables only for virtualenv

- `set -u` fails script with error `./venv/bin/activate: line 57: PS1: unbound variable`. Allowing unbound variable only for this line instead of entire shell script
